docs/references/overview.mdx

```diff
@@ -113,6 +113,13 @@ description: Learn about the Clerk and community SDK's available for integrating
+  
+  ---
+
+  - [SolidJS](https://github.com/spirit-led-software/clerk-solidjs)
+  - Visit the community-maintained repository to learn how to integrate Clerk into your SolidJS application.
+  - {<svg viewBox="0 0 160 178" fill="none"><path fill="url(#f428fb57-d310-4f16-8602-7ef86cb7ba67)" d="M0 55.638v65.737a22.76 22.76 0 0 0 11.386 19.717l56.929 32.873a22.767 22.767 0 0 0 22.771 0l56.929-32.873a22.765 22.765 0 0 0 11.385-19.717V55.638a22.763 22.763 0 0 0-11.385-19.72L91.085 3.053a22.757 22.757 0 0 0-22.77 0l-56.93 32.865A22.77 22.77 0 0 0 0 55.638Z"/><path fill="#fff" d="M39.85 77.129v22.774a5.68 5.68 0 0 0 3.514 5.263 5.685 5.685 0 0 0 2.18.431 5.7 5.7 0 0 0 4.028-1.667 5.693 5.693 0 0 0 1.667-4.027 5.693 5.693 0 1 1 11.385 0 17.081 17.081 0 0 1-29.157 12.077 17.081 17.081 0 0 1-5.002-12.077v-22.77a17.08 17.08 0 0 1 34.159 0 5.693 5.693 0 1 1-11.385 0 5.693 5.693 0 0 0-11.386 0l-.003-.004Zm91.088 22.774a5.693 5.693 0 0 1-5.694 5.694h-5.694v5.691a5.68 5.68 0 0 1-1.668 4.026 5.676 5.676 0 0 1-8.05 0 5.707 5.707 0 0 1-1.667-4.026v-5.691H96.779v5.691a5.682 5.682 0 0 1-5.694 5.694 5.682 5.682 0 0 1-5.69-5.694v-5.691h-5.698a5.695 5.695 0 0 1-5.693-5.693 5.692 5.692 0 0 1 5.693-5.692h5.694V82.826h-5.694a5.692 5.692 0 1 1 0-11.385h5.694v-5.694a5.693 5.693 0 0 1 11.385 0v5.69h11.385v-5.69a5.691 5.691 0 0 1 11.385 0v5.69h5.695a5.68 5.68 0 0 1 4.023 1.669 5.68 5.68 0 0 1 0 8.05 5.692 5.692 0 0 1-4.023 1.67h-5.695v11.386h5.695a5.709 5.709 0 0 1 5.697 5.691Zm-22.773-17.077H96.779v11.386h11.386V82.826Z"/><defs><linearGradient id="f428fb57-d310-4f16-8602-7ef86cb7ba67" x1="15.654" x2="92.51" y1="38.695" y2="172.48" gradientUnits="userSpaceOnUse"><stop stop-color="#927BE5"/><stop offset="1" stop-color="#512BD4"/></linearGradient></defs></svg>}
```

Comment:
Used the SVG above because they dont seem used in the docs, please lemme know if it needs to be something specifically.